src/index.ts

```diff
@@ -824,13 +851,30 @@ export default class Oas {
         },
       })
       .then((dereferenced: RMOAS.OASDocument) => {
+        let circularRefs: string[] = [];
+        if (parser.$refs.circular) {
+          circularRefs = parser.$refs.circularRefs.map(pointer => {
+            // The circular $refs that are coming out of `json-schema-ref-parser` are prefixed
+            // with the schema path (file path, url, whatever) that the schema exists in. Because
+            // we don't care about this information for this reporting mechanism, and only the
+            // $ref pointer, we're removing it.
+            return `#${pointer.split('#')[1]}`;
```

Comment:
Example pointer coming back from `json-schema-ref-parser` looks like `/path/to/file.yaml#/components/schemas/name`. We only need the latter part so that's what this is doing here.